From bfba696be39a8d4de39df6af94724d7017d37905 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Eil=C3=ADs=20N=C3=AD=20Fhlannag=C3=A1in?=
 <pidge@toganlabs.com>
Date: Thu, 25 Feb 2021 17:55:23 +0000
Subject: [PATCH] Remove absolute paths to avoid build host contamination
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Based off of work from Mattias Waldo

Signed-off-by: Eilís Ní Fhlannagáin <pidge@toganlabs.com>
---
 platforms/apsn/cp2112_util/Makefile.am     | 2 +-
 platforms/apsn/tofino_spi_util/Makefile.am | 2 +-
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/platforms/apsn/cp2112_util/Makefile.am b/platforms/apsn/cp2112_util/Makefile.am
index 08749a5..5c090c7 100644
--- a/platforms/apsn/cp2112_util/Makefile.am
+++ b/platforms/apsn/cp2112_util/Makefile.am
@@ -1,6 +1,6 @@
 bin_PROGRAMS = cp2112_util
 cp2112_util_CFLAGS = $(AM_CFLAGS) $(BFPLATFORMS_CFLAGS)
-cp2112_util_CPPFLAGS = -I$(includedir) -I$(top_srcdir)/drivers/include/ -I$(top_srcdir)/platforms/include -I$(top_srcdir)/platforms/apsn/include/ -DDEVICE_IS_ASIC
+cp2112_util_CPPFLAGS = -I$(top_srcdir)/drivers/include/ -I$(top_srcdir)/platforms/include -I$(top_srcdir)/platforms/apsn/include/ -DDEVICE_IS_ASIC
 
 cp2112_util_SOURCES = \
 cp2112_util.c \
diff --git a/platforms/apsn/tofino_spi_util/Makefile.am b/platforms/apsn/tofino_spi_util/Makefile.am
index 8e3a642..4ea269c 100644
--- a/platforms/apsn/tofino_spi_util/Makefile.am
+++ b/platforms/apsn/tofino_spi_util/Makefile.am
@@ -1,6 +1,6 @@
 bin_PROGRAMS = spi_i2c_util
 spi_i2c_util_CFLAGS = $(AM_CFLAGS) $(BFPLATFORMS_CFLAGS)
-spi_i2c_util_CPPFLAGS = -I$(includedir) -I$(top_srcdir)/drivers/include/ -I$(top_srcdir)/platforms/include -I$(top_srcdir)/platforms/apsn/include/ -DDEVICE_IS_ASIC
+spi_i2c_util_CPPFLAGS = -I$(top_srcdir)/drivers/include/ -I$(top_srcdir)/platforms/include -I$(top_srcdir)/platforms/apsn/include/ -DDEVICE_IS_ASIC
 
 spi_i2c_util_SOURCES = \
 tofino_spi_if.c \
-- 
2.30.0

